.github/workflows/cut-release-branch.yml

```diff
@@ -125,20 +122,33 @@ jobs:
           message="New branch ${{ inputs.new_branch_name }} created"
           echo "::notice title="New branch created": $title::$message"
 
-      # TODO: need to add scripts into the adapters to automate bumping the core 
-      # dependency on the `*.latest` branch.  The script should also bump the 
-      # nightly release test branch to the new `*.latest` branch
-      # - name: "Bump dependencies via script"
-      #   # This bumps the dependency on dbt-core in the adapters
-      #   if: ${{ !contains(github.repository, 'dbt-core') }}
-      #   run: |
-      #     echo ${{ github.repository }}
-      #     echo "running some script yet to be written now"
-      #     scripts/update_dependencies.sh
-      #     git status
-      #     git add .
-      #     git commit -m "$commit_message"
-      #     git push
+      - name: "Bump dependencies via script"
+        # This bumps the dependency on dbt-core in the adapters
+        if: ${{ !contains(github.repository, 'dbt-core') }}
+        run: |
+          echo ${{ github.repository }}
+          echo "running update_dependencies script"
+          chmod +x ${GITHUB_WORKSPACE}/.github/scripts/update_dependencies.sh
+          ${GITHUB_WORKSPACE}/.github/scripts/update_dependencies.sh ${{ inputs.new_branch_name }}
+          commit_message="bumping .latest branch variable in update_dependencies.sh to ${{ inputs.new_branch_name }}"
+          git status
+          git add .
+          git commit -m "$commit_message"
+          git push
+
+      - name: "Bump env variable via script"
```

Comment:
should we make execution conditional on `update_release_branch.sh` existing in the repo?

Reply:
I believe all of our adapters will have this part of the action once i get redshift and this action where we want it (currently not able to get `Ensure New Branch Does Not Exist` part to trigger failure if branch already exists, but the second part `Bump dependencies via script` probably should have a conditional added because `spark` won't have this for now